src/components/common/UploadDocumentEwallet.tsx

```
@@ -1,108 +1,467 @@
-				{ type: 'JWT_TOKEN', payload: jwtToken }, // Message data
-				VITE_EWALLET_IFRAME_SRC // Replace with the iframe app's domain
+
+	// Function to close wallet UI
+	const closeWalletUI = () => {
+		setIsIframeVisible(false);
+	};
+
+	// Send authentication data to iframe
+	const sendAuthToIframe = () => {
+		if (!iframeRef.current) return;
+		
```

ESLint

[error] Delete `↹↹`

(prettier/prettier)